modules/custom/openy_repeat/src/RepeatManager.php

```diff
@@ -75,8 +68,7 @@ class RepeatManager implements SessionInstanceManagerInterface {
   /**
    * Constructor.
    */
-  public function __construct(EntityTypeManagerInterface $entity_type_manager, QueryFactory $entity_query, LoggerChannelFactoryInterface $logger_factory, ConfigFactoryInterface $configFactory) {
-    $this->entityQuery = $entity_query;
+  public function __construct(EntityTypeManagerInterface $entity_type_manager, LoggerChannelFactoryInterface $logger_factory, ConfigFactoryInterface $configFactory) {
```

Comment:
@AleksandrPrykhodkoDev you have removed the entity query dependency from the constructor, but I don't see where did you update the service definition for the class?